From eec8c7b8cf9cbec7e217e61335c74a76e3686b51 Mon Sep 17 00:00:00 2001
From: Michal Vasilek <michal@vasilek.cz>
Date: Sat, 20 Aug 2022 23:51:22 +0200
Subject: [PATCH] Fix cross compilation

102e3cef58eb796dd6b3bb099ed0103bcc3d9c75 introduced a check for LMID in
libc to fix compilation on musl, but it used AC_RUN_IFELSE even though
only a compile check is needed. AC_RUN_IFELSE is not available with
cross compilation.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index c83af6a..72bafdf 100644
--- a/configure.ac
+++ b/configure.ac
@@ -76,7 +76,7 @@ AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
        [AC_MSG_RESULT([no])])
 
 AC_MSG_CHECKING([whether dlopen supports LMID])
-AC_RUN_IFELSE([AC_LANG_PROGRAM([
+AC_COMPILE_IFELSE([AC_LANG_PROGRAM([
 			#define _GNU_SOURCE
 			#include <link.h>
 			#include <dlfcn.h>
